spec/models/agent_log_spec.rb

```diff
@@ -11,7 +11,7 @@
     it "requires an agent" do
       @log.agent = nil
       expect(@log).not_to be_valid
-      expect(@log).to have(1).error_on(:agent)
+      expect(@log).to have_at_least(1).error_on(:agent)
```

Comment:
I didn't because we want it to be not optional (that's whats the presence validation is ensuring). I'll found a fix for `delayed_job_active_record` and will apply it in a different PR that will clean up this one :)